app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -145,7 +150,7 @@ class TopicFragmentTest {
     launchTopicActivityIntent(internalProfileId, FRACTIONS_TOPIC_ID).use {
       onView(withId(R.id.topic_tabs_viewpager)).check(matches(isDisplayed()))
       onView(withId(R.id.topic_tabs_viewpager)).perform(swipeLeft())
-      matchTabTitleAtPosition(position = 1)
+      verifyTabTitleAtPosition(position = 1)
```

Comment:
Shouldn't this be ``position = LESSON_TAB_POSITION``? Ditto elsewhere for tab positions.

Reply:
ah, done